src/cplus_plugin/main.py

```diff
@@ -38,15 +43,28 @@ def __init__(self, iface):
 
         # Declare instance attributes
         self.actions = []
-        self.menu = self.tr("&CPLUS")
         self.pluginIsActive = False
+
+        self.menu = QMenu("&Trends.Earth")
```

Comment:
We should use 'CPLUS' here.

Reply:
@Samweli Thanks for noting this! Based the options on that of the trends.earth plugin, likely missed this! Its been fixed.